plugins/Morpheus/stylesheets/ui/_modal.less

```diff
@@ -0,0 +1,20 @@
+.modal {
+  background: @theme-color-background-base;
+}
+
+.modal.open {
+    display: flex !important;
```

Comment:
ℹ️ `!important` is here to override the `display: block` set as inline styles by the JavaScript library